### PR TITLE
Union and intersection

### PR DESCRIPTION
**Pull request by SwarnimaShishodia**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Description

(Union and Intersection of array)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#3647)

### CODE REVIEW

File: C-Plus-Plus/other/Union_and_Intersection_of_arrays.cpp [hunk 1]
```diff
@@ -0,0 +1,154 @@
+using namespace std;
```
Comment (HarshCasper) [inline C-Plus-Plus/other/Union_and_Intersection_of_arrays.cpp:5]: Add a beginner friendly description

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Tiny change. Rest looks good to merge!

**Comment by SwarnimaShishodia** [comment]
I have done the changes.

File: C-Plus-Plus/other/Union_and_Intersection_of_arrays.cpp (hunk 1 above)
Comment (SwarnimaShishodia) [reply C-Plus-Plus/other/Union_and_Intersection_of_arrays.cpp:5]: Done

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Comment by SwarnimaShishodia** [comment]
> Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs
> 
> It would be much appreciated. Thanks!

I will do.

**Pull request merged**